### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Komal-99**
Updated readme for affine transformation

### CODE REVIEW

File: AffineTransformation/README.md [hunk 1]
```diff
@@ -1 +1,89 @@
...
+### Disadvantages<br>
+
+
+### Applications <br>
+* Affine transformation is a linear mapping method that preserves points, straight lines, and planes. Sets of parallel lines remain parallel after an affine transformation. The affine transformation technique is typically used to correct for geometric distortions or deformations that occur with non-ideal camera angles.<br>
+* Affine Transformation helps to modify the geometric structure of the image, preserving parallelism of lines but not the lengths and angles. It preserves collinearity and ratios of distances.<br>
+* This technique is also used to correct Geometric Distortions and Deformations that occur with non-ideal camera angles.<br>
+
+
+### References <br>
```
Comment (geekquad) [inline AffineTransformation/README.md:85]: Add references in bullet points.

Comment (Komal-99) [reply AffineTransformation/README.md:85]: Okay

**Review by geekquad — APPROVED** [review]
Good work @Komal-99.

**Comment by Komal-99** [comment]
Thanks :)

**Pull request merged**